src/commands/trialApp/importTrialApp.ts

```diff
@@ -30,7 +30,7 @@ export async function importTrialApp(_context: IActionContext, loginSession: str
             expirationDate: expirationDate,
             loginSession: loginSession
         };
-
+        context.telemetry.properties.trialTimeRemaining = String(trialAppNode.metadata.timeLeft);
```

Comment:
I feel like you might as well just use the addTrialAppTelemetry here too even though it's obviously going to be a trialApp and they _should_ not be logged in

Reply:
Ah but whatever, it doesn't matter.  Could just be cluttering up the properties too

Reply:
And there's no node, never mind lol sorry for all the comments